__tests__/ReactTags.test.tsx

```diff
@@ -35,6 +35,9 @@ beforeAll(() => {
       { className: '', id: 'Peach', text: 'Peach' },
     ],
     handleDrag: handleDragStub,
+    ariaAttrs: {
+            labelledById:''
+          }
```

Comment:
looks like not formatted

Reply:
formatted now https://github.com/react-tags/react-tags/blob/7fbd38bde1a6b3be2e592b6516e16a330fa05f9e/__tests__/ReactTags.test.tsx#L38